File: flappy_goud/GameConstants.cs
// GameConstants.cs

public static class GameConstants
{


    // TODO: Changing TargetFps will adjust the game speed. We need to see if that is a CS issue or Rust issue
    public const uint TargetFPS = 120;
    public const uint ScreenWidth = 288;
    public const uint ScreenHeight = 512;

    public const float Gravity = 9.8f;
    public const float JumpStrength = -3.5f;
    public const float JumpCooldown = 0.33f;

    public const float PipeSpeed = 1.0f;
    public const float PipeSpawnInterval = 1.5f;
    public const int PipeWidth = 60;
    public const int PipeGap = 100;

}

File: examples/ios/flappy_bird/Sources/FlappyBird/GameConstants.swift
// GameConstants.swift -- iOS Flappy Bird
// Portrait orientation: 288x512 game area + 112 base

import Foundation

enum GameConstants {
    static let targetFPS: UInt32 = 120
    static let baseHeight: Float = 112

    // Portrait dimensions matching the original Flappy Bird aspect ratio
    static let screenWidth: UInt32 = 288
    static let screenHeight: UInt32 = 512
    static let gravity: Float = 9.8
    static let jumpStrength: Float = -3.5
    static let jumpCooldown: Float = 0.30

    static let pipeSpeed: Float = 1.0
    static let pipeSpawnInterval: Float = 1.5
    static let pipeWidth: Float = 60
    static let pipeGap: Float = 100
}
